torchvision/transforms/v2/_meta.py

```diff
@@ -63,6 +71,12 @@ def _transform(
 
 
 class ClampBoundingBox(Transform):
+    """[BETA] Clamp bounding boxes coordinates between ``[0, spatial_size[0]]`` for x-axis and
+    ``[0, spatial_size[1]]`` for y-axis.
```

Comment:
The description is a bit low-level, I'd suggest something higher-level like

```suggestion
    """[BETA] Clamp bounding boxes to their corresponding image dimensions.
    
    The clamping is done according to the bounding boxes' ``spatial_size`` meta-data.
```